streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/FlinkSqlController.java

```diff
@@ -79,6 +81,18 @@ public RestResponse verify(String sql, Long versionId, Long teamId) {
     }
   }
 
+  @PostMapping("list")
+  public RestResponse list(Long appId, RestRequest request) {
+    IPage<FlinkSql> page = flinkSqlService.page(appId, request);
+    return RestResponse.success(page);
+  }
+
+  @PostMapping("delete")
```

Comment:
@zhoulii Thanks for the contribution.

Does this operation need permission?

Reply:
you are right, I will add permission for this.